src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -221,7 +220,13 @@ private void ProcessRequests(ITestRequestManager testRequestManager)
 
                         case MessageType.CustomTestHostLaunchCallback:
                             {
-                                this.onAckMessageReceived?.Invoke(message);
+                                this.onCustomTestHostLaunchAckReceived?.Invoke(message);
+                                break;
+                            }
+
+                        case MessageType.ProxyAttachDebuggerToProcessCallback:
```

Comment:
>ProxyAttachDebuggerToProcessCallback [](start = 41, length = 36)

Can we make it simpler? How about `AttachDebuggerCallback`

Reply:
Agreed, we can indeed simplify this but not by that much. This (`ProxyAttachDebuggerToProcessCallback`) is the callback the IDE uses to call into vstest.console, however there's another callback that we may want to change as well, i.e. the one that vstest.console uses to call into testhost (`AttachDebuggerToProcessCallback`).

It's imperative we use two different messages, so how about `ConsoleAttachDebuggerCallback` for the first one and `TesthostAttachDebuggerCallback` for the last one ?